pastis/fields_building_numerical_all_laurent.py

```diff
@@ -0,0 +1,445 @@
+"""
+This module contains functions that calculates the field sensitivities for a given coronagraph design
+"""
+
+import os
+import time
+import functools
+from shutil import copy
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.io import fits
+import astropy.units as u
+import hcipy as hc
+
+from config import CONFIG_INI
+import util_pastis as util
+from e2e_simulators.luvoir_imaging_onephot import LuvoirAPLC
```

Comment:
These imports are broken and need to be adjusted, I think they're just missing a `pastis.` in front of them.